pixi.toml

```diff
@@ -0,0 +1,50 @@
+[project]
+authors = ["IRIS-HEP"]
+channels = ["conda-forge"]
+description = "Add a short description here"
+name = "analysis-grand-challenge"
+platforms = ["linux-64", "osx-64", "osx-arm64"]
+version = "1.4.0"
+
+[tasks]
+
+[dependencies]
+python = ">=3.7"
+notebook = ">=7.2.2"
+jupyterlab = ">=4.2.5"
+pixi-kernel = ">=0.5.1"
+ipykernel = ">=6.29.5"
+pre-commit = ">=4.0.1"
+jupytext = ">=1.16.4"
+
+[pypi-dependencies]
+servicex = ">=3.0.0"
+
+[feature.latest.dependencies]
+cabinetry = ">=0.6.0"
+scikit-learn = ">=1.0"
+xgboost = ">=1.0"
+coffea = ">=2024.9.0"
+
+[feature.cms-open-data-ttbar.tasks]
+start = "jupyter lab"
+
+# Note: This is global as using '--user'
+install-ipykernel = """
+python -m ipykernel install --user --name="cms-open-data-ttbar" --display-name="cms-open-data-ttbar"
+"""
+
+[feature.cms-open-data-ttbar.dependencies]
+vector = ">=1.5.2"
+cabinetry = ">=0.6.0"
+scikit-learn = ">=1.0"
+# coffea-casa restrictions
+python = ">=3.7,<3.10"
+coffea = ">=0.7.22,<1.0"
+xgboost = ">=1.0,<1.8"
+dask = "2023.11.0.*"
+cloudpickle = "3.0.0.*"
```

Comment:
These constratnts still differ from the existing coffea-casa image used on the runner

```
VersionMismatchWarning: Mismatched versions found

+---------+----------------+----------------+----------------+
| Package | Client         | Scheduler      | Workers        |
+---------+----------------+----------------+----------------+
| lz4     | 4.3.3          | 4.3.2          | 4.3.2          |
| msgpack | 1.1.0          | 1.0.6          | 1.0.6          |
| pandas  | 2.2.3          | 2.1.2          | 2.1.2          |
| python  | 3.9.20.final.0 | 3.9.18.final.0 | 3.9.18.final.0 |
| toolz   | 1.0.0          | 0.12.0         | 0.12.0         |
| tornado | 6.4.1          | 6.3.3          | 6.3.3          |
+---------+----------------+----------------+----------------+
```

but it seems to work without errors.